app/components/employees.tsx

```diff
@@ -0,0 +1,76 @@
+import EmployeeCard from "./employeeCard";
+import styles from "./home.module.scss";
+import employeeStyles from "./employees.module.scss";
+import { EmployeeItemProp } from "../function/Employees";
+import { useChatStore } from "../store";
+import { isMobileScreen } from "../utils";
+import { useState } from "react";
+import { IconButton } from "./button";
+import ChatIcon from "../icons/chat.svg";
+import SettingsIcon from "../icons/settings.svg";
+import Link from "next/link";
+
+type EmployeesProps = {
+  showSideBar?: (newState: boolean) => void;
+} & EmployeeItemProp;
+
+export default async function Employees({
+  employees,
+  showSideBar,
+}: EmployeesProps) {
+  const config = useChatStore((state) => state.config);
+  const [selectedOffice, setSelectedOffice] = useState("");
+
+  return (
+    <>
+      <div className={styles["window-header"]}>
+        <div className={styles["window-header-title"]}>
+          <div className={styles["window-header-main-title"]}>Ansatte</div>
+          <div className={styles["employee-buttons"]}>
+            <div>
+              <button onClick={() => setSelectedOffice("Trondheim")}>
```

Comment:
Her kan det kanskje heller brukes en map for å unngå duplikat kode?

Reply:
Det skal jeg gjøre!